tests/git-scripts/pre-push.sh

```diff
@@ -32,18 +32,9 @@ echo "URL is $URL"
 
 set -eu
 
-run_test()
-{
-    TEST=$1
-    echo "running '$TEST'"
-    if ! `$TEST > /dev/null 2>&1`; then
-        echo "test '$TEST' failed"
-        return 1
-    fi
+# shorten output but be sure to catch any error
+not() {
+    ! "$@"
 }
-
-run_test ./tests/scripts/check-doxy-blocks.pl
-run_test ./tests/scripts/check-names.sh
-run_test ./tests/scripts/check-generated-files.sh
-run_test ./tests/scripts/check-files.py
-run_test ./tests/scripts/doxygen.sh
+{ tests/scripts/all.sh -k 'check_*' || echo "FAILED all.sh"; } 2>&1 |
+    not grep FAILED
```

Comment:
If something fails, the output will be something like `FAILED: 1`. It would be more informative to show what failed. For example by grepping or sedding lines surrounded by `^^^^`.

Reply:
This question is no longer relevant with the new `all.sh -q` approach, so I'm marking the conversation as resolved.